### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by Kasia9311**
Adding Fibonacci

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Kasia9311

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Kasia9311 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -2,10 +2,24 @@
 
 int fibonacci_iterative(int sequence) {
     // TODO: Your implementation goes here
-    return 0;
+    if (sequence < 2){
+    	return sequence;
+    }
+    int a = 1;
+    int b = 1;
+    while (sequence--> 2){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:10]: Add space.  `-->` Is really confusing

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -2,10 +2,24 @@
...
+    	b += a;
+    	a = b - a;
+    }
+    return b;
 }
 
 int fibonacci_recursive(int sequence) {
     // TODO: Your implementation goes here
-    return 0;
+    if (sequence <=1){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:19]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -2,10 +2,24 @@
...
+    	return sequence;
+    } 
+    
+    return fibonacci_recursive(sequence-1) + fibonacci_recursive(sequence-2);
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:23]: Add space between operations like '+' '-' '!=' '==' '%' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**